iphone/cli/commands/_build.js

```diff
@@ -6821,14 +6867,18 @@ iOSBuilder.prototype.invokeXcodeBuild = function invokeXcodeBuild(next) {
 		'SYMROOT=' + path.join(this.buildDir, 'build', 'Products'),
 	];
 
-	if (this.simHandle) {
+	if (this.simHandle && this.target !== 'mac') {
 		args.push('-destination', 'generic/platform=iOS Simulator');
 
 		// only build active architecture, which is 64-bit, if simulator is not 32-bit (iPhone 5s or newer, iPhone 5 and older are not 64-bit)
 		if (this.simOnlyActiveArch) {
 			args.push('ONLY_ACTIVE_ARCH=1');
 		}
 	}
+	
+	if (this.target === 'mac') {
+	   args.push('SUPPORTS_MACCATALYST=YES');
```

Comment:
<!--
  1 failure:  iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_build.js;
  Line: 6880;
-->
- :no_entry_sign: iphone/cli/commands/_build.js line 6880 – Mixed spaces and tabs. (no-mixed-spaces-and-tabs)